src/components/DropZone/DropZone.tsx

```diff
@@ -304,30 +307,58 @@ export class DropZone extends React.Component<CombinedProps, State> {
 
     this.setState({error: this.props.error});
 
-    if (!this.dropNode) {
+    const dropNode: HTMLDivElement | Document | null =
+      'current' in this.dropNode ? this.dropNode.current : document;
+
+    if (!dropNode) {
       return;
     }
 
-    addEventListener(this.dropNode, 'drop', this.handleDrop);
-    addEventListener(this.dropNode, 'dragover', this.handleDragOver);
-    addEventListener(this.dropNode, 'dragenter', this.handleDragEnter);
-    addEventListener(this.dropNode, 'dragleave', this.handleDragLeave);
+    addEventListener(dropNode as EventTarget, 'drop', this.handleDrop);
```

Comment:
Any idea why we have to assert the type? Ideally we shouldn't have to force it.

Reply:
`[ts]
Argument of type 'HTMLDivElement | Document | null' is not assignable to parameter of type 'EventTarget'.
  Type 'null' is not assignable to type 'EventTarget'. `

I guess this has to do with the definitions of the event listeners in `@shopify/javascript-utilities/events` which requires that the first argument be of type _EventTarget_.

Any other workarounds? 😄